dotCMS/src/main/java/com/dotcms/security/secret/Secret.java

```diff
@@ -0,0 +1,44 @@
+package com.dotcms.security.secret;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class Secret {
+
+    private final char[] value;
+    private final boolean hidden;
+    private final SecretType secretType;
+
+    private Secret(final char[] value, final SecretType secretType, final boolean hidden) {
+        this.value = value;
+        this.secretType = secretType;
+        this.hidden = hidden;
+    }
+
+    @JsonIgnore
+    public String getString() {
+        return String.valueOf(value);
+    }
+
+    @JsonIgnore
+    public boolean getBoolean() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A getX() method which returns a boolean should be named isX()](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4623512)